api/grpcserver/mesh_service.go

```diff
@@ -147,11 +135,9 @@ func (s MeshService) getFilteredActivations(startLayer types.LayerID, addr types
 		return nil, status.Errorf(codes.Internal, "error retrieving activations data")
 	}
 	for _, atx := range atxs {
-		// Filter here, now that we have full data
-		if atx.Coinbase == addr {
-			activations = append(activations, atx)
-		}
+		activations = append(activations, atx)
```

Comment:
This loop and this new array are now redundant, it's just copying the existing `atxs` array

Reply:
atxs variable is a map. Just did a quick google search, I wasn't able to find a more efficient way of extracting all values from a map into a slice/array